generate-protos.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
```

Comment:
Because the proto definitions for ACLed artifacts APIs are not available in older versions of MLflow, we include `databricks_artifacts.proto` and its dependencies (`databricks.proto` / `scalapb.proto`), as well as this script for compiling them into python classes (based on https://github.com/mlflow/mlflow/blob/master/generate-protos.sh).